docs/form.component.md

```diff
@@ -235,6 +235,197 @@ also implement your own custom validators to replace or extend the set. See the
+                                "sizeY": 1,
+                                "row": -1,
+                                "col": -1,
+                                "visibilityCondition": null,
+                                "endpoint": null,
+                                "requestHeaders": null
+                            },
+                            {
+                                "fieldType": "FormFieldRepresentation",
+                                "id": "Date",
+                                "name": "Date",
+                                "type": "date",
+                                "value": null,
+                                "required": false,
+                                "readOnly": false,
+                                "overrideId": false,
+                                "colspan": 1,
+                                "placeholder": null,
+                                "minLength": 0,
+                                "maxLength": 0,
+                                "minValue": null,
+                                "maxValue": null,
+                                "regexPattern": null,
+                                "optionType": null,
+                                "hasEmptyValue": null,
+                                "options": null,
+                                "restUrl": null,
+                                "restResponsePath": null,
+                                "restIdProperty": null,
+                                "restLabelProperty": null,
+                                "tab": "tab1",
+                                "className": null,
+                                "params": {
+                                    "existingColspan": 1,
+                                    "maxColspan": 2
+                                },
+                                "dateDisplayFormat": null,
+                                "layout": {
+                                    "row": -1,
+                                    "column": -1,
+                                    "colspan": 1
+                                },
+                                "sizeX": 1,
+                                "sizeY": 1,
+                                "row": -1,
+                                "col": -1,
+                                "visibilityCondition": null
+                            }
+                        ]
+                    }
+                }
+            ],
+            "outcomes": [],
+            "javascriptEvents": [],
+            "className": "",
+            "style": "",
+            "customFieldTemplates": {},
+            "metadata": {},
+            "variables": [],
+            "customFieldsValueInfo": {},
+            "gridsterForm": false,
+            "globalDateFormat": "D-M-YYYY"
+        }
+```
+
+Below component with the form definition JSON assigned to the variable `formDefinitionJSON`, shows how a form is rendered by using form definition JSON.
```

Comment:
It's clearer if you say "The component below with the form definition..." instead of "Below component with...".